api/v2alpha1/pipeline_spec.proto

```diff
@@ -732,7 +732,7 @@ message PipelineDeploymentConfig {
     // API Specification for invoking a Google Cloud AI Platform CustomJob.
     // The fields must match the field names and structures of CustomJob
     // defined in
-    // https://cloud.google.com/ai-platform-unified/docs/reference/rest/v1beta1/projects.locations.customJobs.
+    // https://cloud.google.com/ai-platform-unified/docs/reference/rest/v1/projects.locations.customJobs.
```

Comment:
will undo this

Reply:
I'm thinking can we keep both v1beta1 and v1 endpoints so that we can have backward compatibility while introducing the new v1 endpoint. what do you think.

Reply:
Thanks, please correct me, currently we do not have either api/v1beta1 or api/v1, it is just api/ . So backward compatibility with what?

Reply:
Do you think we should have separate code paths v1 and v1beta1 : https://github.com/kubeflow/kfp-tekton/pull/1085/commits/cba2fa66936776b49f6afa71053d0c482ff5f089

Reply:
Yes, I think the `api/v1beta1` is just refactor of the current `/api`. So we just want to add another `api/v1` code path that creates the /v1 REST/GRPC endpoints.